lib/crypto/butil.js

```diff
@@ -40,12 +40,21 @@ module.exports.unbuffer = function unbuffer(object, inplace) {
   return copy
 }
 
-module.exports.bufferize = function bufferize(object, inplace) {
+module.exports.bufferize = function bufferize(object, inplace, exceptions) {
```

Comment:
Also `exceptions` triggers certain uncomfortable feelings in some people (me) because of its specific form of torture in other languages. Perhaps `ignore` or something?